openapi/src/main/kotlin/com/expediagroup/sdk/generators/openapi/OpenApiSdkGenerator.kt

```diff
@@ -128,6 +128,14 @@ class OpenApiSdkGenerator {
 
     private fun getSdkLanguage() = if (isKotlin.toBoolean()) "kotlin" else "java"
 
+    private fun preProcessSpecFile(path: String): String {
```

Comment:
Shouldn't we move this logic to the `YamlProcessor` class? we could move the logic there and remove this method. 

(There is not a lot of value in exposing the `unifyTags` and `dump` methods here. Just call the `YamlProcessor` and let it do its thing).

Reply:
This is to allow for passing the tag value we want to replace the other tags, as well as the file (temp file) that everything should be dumped in.
They should be external as their values are used externally and are injected from there.
Exposing these methods would allow for future customization of how to pre-process the file and which steps to use depending on external flags.

Reply:
You could pass them in the constructor as you do with the path
``` val yamlProcessor = YamlProcessor(path, namespace)```

-  the tempFile is already being generated, no need to pass it - or you can pass it in the constructor too if you feel it could be customized. _I would not pass it for now_.
- you can do any conversion to the namespace inside the `YamlProcessor` when needed (e.g., camelCase).